andela/odin-ah-backend
# #160452470 Integrate Coveralls code coverage service

#### What does this PR do?
Integrate Coveralls code coverage service
#### Description of Task to be completed?
- Link coveralls to the github repo
- Add coveralls badge to readme file
- Configure travis to run coverage after test has passed
#### How should this be manually tested?
- Clone repo or pull latest changes
- Checkout to `ch-160452470-coveralls-setup`
```bash
> git checkout ch-160452470-coveralls-setup
```
- Install packages 
```bash 
 > npm install
```
- Run coverage script 
```bash
> npm run coverage
```
- Coverage report should be logged on the command line
- [Coveralls build output](https://coveralls.io/builds/19222433)
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
[#160452470](https://www.pivotaltracker.com/story/show/160452470)
#### Screenshots (if appropriate)
- N/A
#### Questions:
- None
